server/game/cardaction.js

```diff
@@ -20,6 +20,7 @@ class CardAction {
     constructor(game, card, properties) {
         this.game = game;
         this.card = card;
+        this.canPlayFrom = properties.canPlayFrom || 'play';
```

Comment:
Following convention from elsewhere, the default should be 'play area' or the convention should be changed to use 'play' elsewhere depending on what @cryogen thinks. The case I'm aware of is `targetLocation` on effects, which uses 'play area' to refer to everything that's face up on the board, which includes (like this case), 'play area', 'agenda' and 'active plot' board locations.